### PR TITLE
maintenance: minor README updates

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Includes a few minor updates to the primary README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ root
 │   ├── <a href="./packages/batch-submitter">batch-submitter</a>: Service for submitting batches of transactions and results to L1
 │   ├── <a href="./packages/message-relayer">message-relayer</a>: Tool for automatically relaying L1<>L2 messages in development
 │   └── <a href="./packages/replica-healthcheck">replica-healthcheck</a>: Service for monitoring the health of a replica node
+├── <a href="./go">go</a>
+│   ├── <a href="./packages/batch-submitter">batch-submitter</a>: Service for submitting batches of transactions and results to L1
```
Comment (tynes) [inline README.md:42]: This should be `./go/batch-submitter` right?

Comment (smartcontracts) [reply README.md:42]: Oops, good catch

**Review by cfromknecht — APPROVED** [review]
LGTM 🎉

**Comment by cfromknecht** [comment]
@smartcontracts needs a quick rebase

**Pull request merged**